src/jekyll/_plugins/custom_tags/include_code.rb

```diff
@@ -151,18 +151,18 @@ def render_codehighlighter(context, code, filepath)
 
       link_text = site.data["localized_strings"][page["langcode"]]["try_sample"]
       link_text = link_text ? link_text : site.data["localized_strings"]["en"]["try_sample"]
+      #TODO: Move the HTML snippet below into a liquid component so it can
```

Comment:
Slightly fearful that these TODO's should be issues rather than comments in the code.